src/config/config.testnet.ts

```diff
@@ -46,10 +34,9 @@ export const networks: NetworkType[] = [
     name: 'Sovereign',
     apiAddress: 'https://api-sovereign-test.elrond.ro',
     gatewayUrl: '',
-    extrasApi: 'https://extras-api.multiversx.com',
+    extrasApi: 'https://extras-api-sovereign-test.elrond.ro',
```

Comment:
From what @radumojic told me yesterday, is it alright to have these endpoints public?